### ISSUE TITLE
Run golden tests for some other repos as part of CI in abc repo

### ISSUE THREAD

**Issue by drevell** [abcxyz/abc#404]
### TL;DR

There are some important templates outside of the abc repo that we should be careful not to break. I'm thinking about:

 - abcxyz/gcp-org-terraform-template
 - abcxyz/flutter-demo
 - ‎abcxyz/github-metrics-aggregator‎
 - ‎abcxyz/infra-gcp‎
 - ‎abcxyz/jvs‎
 - ...anything else that has a spec.yaml

In the CI process for the abc repo (lint-test.yml), let's checkout these repos and run all their golden tests. This will help detect regression in the abc CLI before they're shipped.

Some ideas for how this could look:

 - We could write a shell script that does a `git checkout` of each of the above repos then runs `abc templates render golden-test verify` for each one. We should wait until #311 is implemented so we can easily run every golden test in the repo.
   - Extra credit: do each repo concurrently so we're not waiting for a long time.

We'll also need to actually *create* golden tests for the above repos using `abc templates render new-test`.

Tentatively assigning to @capri-xiyue who's working on golden tests.

cc @verbanicm who has an interest in some of the above repos
